### PR TITLE
[Header-section] -Created header section.

### PR DESCRIPTION
**Pull request by mamatkulov95**
image
Created Header section.

### CODE REVIEW

File: src/Header/Header.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  width: 50px;
+  height: 45px;
+  align-self: center;
+}
+
+.header-left-text {
+  font-family: 'Karla', sans-serif;
+  font-weight: 700;
+  font-size: 25px;
+  line-height: 24px;
```
Comment (abanseka) [inline src/Header/Header.css]: - [x]  -  `line-height` has no use here. please remove it.

File: src/Header/Header.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.header-logo {
+  width: 50px;
+  height: 45px;
+  align-self: center;
+}
+
+.header-left-text {
+  font-family: 'Karla', sans-serif;
```
Comment (abanseka) [inline src/Header/Header.css:22]: - [x]  - I think you need to add the `Karla` font. to the project

**Review by abanseka — CHANGES_REQUESTED** [review]

File: src/Header/Header.css (hunk 1 above)
Comment (mamatkulov95) [reply src/Header/Header.css]: Resolved

File: src/Header/Header.css (hunk 2 above)
Comment (mamatkulov95) [reply src/Header/Header.css:22]: Added

**Review by abanseka — APPROVED** [review]

**Pull request merged**